leadpages.net-prod.custom-domain.json

```diff
@@ -0,0 +1,17 @@
+{
+   "providerId": "leadpages.net-prod",
+   "providerName": "Leadpages",
+   "serviceId": "custom-domain",
+   "serviceName": "Leadpages Custom Domain",
+   "version": 1,
+   "logoUrl": "https://my.leadpages.net/static/current/img/logos/logo_icon.svg",
+   "description": "Enables a domain to work with Leadpages",
+   "variableDescription": "DESTINATION: The subdomain for the domain.",
+   "destinationRequired": true,
+   "records": [{
+      "type": "CNAME",
+      "host": "%DESTINATION%",
```

Comment:
Please rather use "@" or "%host%" here, as this allows you to apply the same template many times without being replaced.
Please refer to this section of spec for details:
https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#65-variables-and-host

in your case, as host is always required (CNAME is not allowed on APEX), you'd also set "hostRequired": true

Generally you can base on this template which is very alike:
https://github.com/Domain-Connect/Templates/blob/master/app.unbounce.com.site.json